content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -394,6 +394,43 @@ NAME              PARENTREF
 2001:db8:1:2::a   services/kube-system/kube-dns
 ```
 
+This new feature also allow users to dynamically define the `service-cluster-ip-range` using
+ServiceCIDR objects. During bootstrap, a default ServiceCIDR object named `kubernetes` is created
+from the flags:
+
+```shell
+kubectl get servicecidrs
+```
+```
+NAME         CIDRS         AGE
+kubernetes   10.96.0.0/28  17m
+```
+
+Users can create or delete new ServiceCIDR objects to add or remove ip ranges in order to increase or reduce the number of virtual cluster IPs
```

Comment:
Why do we delete "new ServiceCIDR" objects ... ?
Please revise the text to avoid confusion.
Please always spell "IP" rather than "ip".

"virtual cluster IPs" reads like a new term.
It is more confusing because there are "virtual clusters" in the ecosystem today.

Reply:
users may want to add a ServiceCIDIR to expand their ranges, but they may want to shrink their ranges too, so they can create a new ServiceCIDR with a smaller range and delete the larger one (they need to recreate the Services to keep all of them within the ServiceCIDR)

I have to do a manual on how to use this, what is the best place to document it, now we are talking I think that is better if I create its own section and explain all of this more in detail

Reply:
What I meant in a previous comment is that we may want to revise the text to make it easier to read. For example,

> Users can create new ServiceCIDR objects to add IP ranges in order to increase the
> number of cluster IPs for Services. They can also delete existing ServiceCIDR objects
> to reduce IP ranges available as cluster IPs for Services.

Reply:
Speaking of the new content, how about adding a task page under `content/en/docs/tasks/network/` ?

Reply:
pushed a new commit with the tasks page, please review and let me know if you have some suggestions on how to better link it from here

Reply:
done